modules/storages/lib/api/v3/file_links/join_origin_status_to_file_links_relation.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module API
+  module V3
+    module FileLinks
+      class JoinOriginStatusToFileLinksRelation
+        # @param [Hash] id_status_map A hash mapping file link IDs to their origin status
+        # in the format { 137: "view_allowed", 142: "error" }
+        def self.create(id_status_map)
+          sanitized_sql = ActiveRecord::Base.send(
```

Comment:
Do you need to use `send`? Can't you just call the method? `sanitize_sql_array([...])`

Reply:
Uh, interesting. I took this code sample from rubocop's improvements. Will try it.

Reply:
works, good catch!